src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Marshaling/JSMarshalerArgument.Object.cs

```diff
@@ -88,7 +88,7 @@ public unsafe void ToManaged(out object? value)
                 }
                 else
                 {
-                    throw new NotImplementedException("ToManaged: " + slot.ElementType+ "[]");
+                    throw new NotImplementedException(SR.Format(SR.ToManagedNotImplemented, slot.ElementType+ "[]"));
```

Comment:
I think all these should be `NotSupportedException` as `NotImplementedException` is for "work in progress" things

Reply:
From my point of view still yes.